plugins/pre-commit.sh

```diff
@@ -46,15 +45,15 @@ fi
 
 
 # Check for unstaged changes to files in the index.
-CHANGED_FILES=(`git diff --name-only ${PYTHON_FILES[@]}`)
-if [ "$CHANGED_FILES" ]; then
+CHANGED_FILES=$(git diff --name-only "${PYTHON_FILES[@]}")
+if [ "${CHANGED_FILES}" ]; then
   echo 'You have unstaged changes to some files in your commit; skipping '
   echo 'auto-format. Please stage, stash, or revert these changes. You may '
   echo 'find `git stash -k` helpful here.'
   echo
   echo 'Files with unstaged changes:'
-  for file in ${CHANGED_FILES[@]}; do
-    echo "  $file"
+  for file in "${CHANGED_FILES[@]}"; do
```

Comment:
you don't need the for loop anymore.  the single printf statement below is sufficient.

Reply:
Att to detail, I missed that 
Thanx!